cpp/src/parquet/file_reader.cc

```diff
@@ -33,6 +33,8 @@
 #include "arrow/util/int_util_overflow.h"
 #include "arrow/util/logging.h"
 #include "arrow/util/ubsan.h"
+#include "parquet/bloom_filter.h"
```

Comment:
This is already included in the bloom_filter_reader.h and can be removed safely here.